packages/vue/src/examples/pages/checkout/_internal/Shipping.vue

```diff
@@ -73,7 +73,7 @@
         v-model="country"
         :value="country"
         label="Country"
-        class="form__element form__element--half form__element--half-even form__select sf-select--underlined"
+        class="form__element form__element--half form__element--half-even form-select sf-select--underlined"
```

Comment:
@AdamPawlinski one small thing, let's keep the BEM style class name for this element and we are good to go with this PR :)

Reply:
I'm sorry, my mistake :)